cli/server_test.go

```diff
@@ -2150,7 +2151,7 @@ func TestConnectToPostgres(t *testing.T) {
 		dbURL, err := dbtestutil.Open(t)
 		require.NoError(t, err)
 
-		sqlDB, err := cli.ConnectToPostgres(ctx, log, "postgres", dbURL, migrations.Up)
+		sqlDB, err := cli.ConnectToPostgres(ctx, log, "postgres", dbURL, migrations.Up, codersdk.DefaultMaxConns, codersdk.DefaultIdleConns)
```

Comment:
Can we also test with different values for Max/Idleconns and assert that this is reflected in `db.Stats()`?

Reply:
Turns out this isn't really feasible :(

Reply:
I didn't add tests because calling `db.Stats()` doesn't return the idle count because we in fact set _max_ idle conns, so the lib doesn't spin up those idle conns, therefore `Stats` doesn't surface them.